packages/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -32,7 +33,8 @@ export interface TransactionObject {
 	readonly senderPublicKey: string;
 	signatures?: ReadonlyArray<string>;
 	readonly type: number;
-	containsUniqueData: boolean;
+	readonly senderId: string;
+	containsUniqueData?: boolean;
```

Comment:
i think this will be there by default?

Reply:
I have made this change because the JSON transaction object do not contain this property. And in order to use fixtures, I have marked this property as optional.

Reply:
addressed in my PR too